packages/packem/src/rollup/plugins/styles/index.ts

```
@@ -0,0 +1,410 @@
+                if (extractedData.map && sourceMap) {
+                    const fileName = this.getFileName(cssFileId);
+
+                    const assetDirectory =
+                        typeof options_.assetFileNames === "string"
+                            ? normalizePath(dirname(options_.assetFileNames))
+                            : typeof options_.assetFileNames === "function"
+                              ? normalizePath(dirname(options_.assetFileNames(cssFile)))
+                              : "assets"; // Default for Rollup v2
+
+                    const map = mm(extractedData.map)
+                        .modify((m) => (m.file = basename(fileName)))
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)